### ISSUE TITLE
Text size on table cells

### ISSUE THREAD

**Issue by itispx** [gomutex/godocx#42]
When upgrading to a newer version I can no longer find a way to set the size of the text inside a table.

This is the code that was previously working:

```go
row := table.AddRow()
row.AddCell().AddParagraph("").AddText("1").Size(tableTextSize)
row.AddCell().AddParagraph("").AddText("2").Size(tableTextSize)
row.AddCell().AddParagraph("").AddText("3").Size(tableTextSize)
row.AddCell().AddParagraph("").AddText("4").Size(tableTextSize)
row.AddCell().AddParagraph("").AddText("5").Size(tableTextSize)
```

I tried to set the `size` property in other ways but did not find a way.

thx for the library btw very useful

**Comment by gomutex** [comment]
The public API get broken when i restructured to get style prop for para. Fixed it in v0.1.4 now

**Comment by itispx** [comment]
you are a fucking legend
